README.md

```diff
@@ -180,25 +210,27 @@ ReactDOM.render(
 );
 ```
 
-# API
+# API Docs
 
 ## SubscriptionClient
 ### `Constructor(url, options, connectionCallback)`
-- `url: string` : url that the client will connect to
+- `url: string` : url that the client will connect to, starts with `ws://` or `wss://`
 - `options?: Object` : optional, object to modify default client behavior
   * `timeout?: number` : how long the client should wait in ms for a subscription to be started (default 5000 ms)how long the client should wait in ms for a subscription to be started (default 5000 ms)
-  * `connectionParams?: Object` : object that will be available as first argument of `onConnect` (in server side)
+  * `lazy?: boolean` : use to set lazy mode - connects only when first subscription created, and delay the socket initialization
+  * `connectionParams?: Object | Function` : object or function that will be available as first argument of `onConnect` (in server side)
```

Comment:
Is this true? Isn't only the connectionParams function used to return the the connectionParams to be sent to the server? The description as is gives the idea that the function is sent 😄